### ISSUE TITLE
Be more active in revel-framework@googlegroups.com

### ISSUE THREAD

**Issue by anonx** [revel/revel#892]
Guys, you forbade asking questions using GH issues where I could answer in minutes or hours. I'm not able to do it via Google Groups.

So, please **be active answering the questions via google groups**. There are tons of unanswered ones: https://groups.google.com/forum/#!forum/revel-framework
Look, 1 month and still no answers.

@brendensoares, @notzippy, @revel.

**Comment by anonx** [comment]
#### Status of the project
It feels like the project is stalling. Reforms are needed. First of all let's find out who's still on board.

Are you still using and/or interested in development of Revel Framework, [Core Team](https://github.com/revel/revel/wiki/Core-Team)?
1. @brendensoares 
2. @pushrax 
3. @notzippy 
4. @landaire 
5. @pedromorgan 
6. I'm dead sure there was somebody else, there were 8 people including me and robfig. verdverm?

**Comment by landaire** [comment]
Unfortunately I'm not able to be nearly as active as I once was which is quite obvious now. Work/school/personal life got too busy.

**Comment by notzippy** [comment]
Yes I am actively developing a large project using revel, I have not
participated much development wise lately because I was waiting for the
restructure to be completed. Brendon has done an excellent job splitting up
the work into sections, but it is difficult to contribute work when we
remain in a wait for restructure state.

Should we open up multiple development branches then? Or would that get to
chaotic? I do not have experience in this matter, my projects have been
solo effort..

**Comment by notzippy** [comment]
Some additional thoughts, I think we also need some roles / responsibilites defined in the core group as well. 

1. First Responder - responsible for looking at new pull requests, bug reports and responding to them. 
2. Development core - responsible for a specific branch, calling feature cut offs and creating new branches
3. Documentation core - Responsible for maintaing / updating the website in responses to releases and changes to the development.
4. Public Relations - Resposible for checking / updating the forums messages, doing the biweekly report on the forum update the community with new releases information, bug fixes, future direction. 
5. Design lead, resonsible for coordinating the development core, documentation core and first responders with bug fixes

I find without some defined roles it is difficult to contribute, or that one person ends up doing most of the work. I would go further to suggest these roles be rotated throughout the core team so that the work is not always on one person. A person can (will need to) belong to multiple roles, but should remain in that role for a minimum of 2 weeks and a max of 2 months. Finally we should also be more transparent within the core group regarding on what we expect to get accomplished and in what time frame.

**Comment by anonx** [comment]
@notzippy, I like your idea of roles/responsibilities and transparency of expectations & time frames.

Well, 1 active and 1 inactive team member.
Should more people be invited? Probably, some active collaborators: https://github.com/revel/revel/graphs/contributors

**Comment by pedromorgan** [comment]
I dont mind looking after the docs, (when i have time)

**Comment by notzippy** [comment]
Yes more are needed, I would look to recent PR for possible contributors,  in theory those are the ones who are active in the project.

**Comment by pedromorgan** [comment]
I have a view on this...

We need to make revel.support in revel...
We need a faq.. powered from example..
a github bugs list.. as an remote api
browse the manual on mobile, tablet or Virtual..

We want to get more back from out community and move faster..

TO achieve this, I think we need to stake more risks with "develop" than current..

and then throw than into a branch called "next" = next expected version = testing

and then into master fast..

I dont think we need too much of a process, with rfc etc..
- we can instead achieve the same things in branches, and the
new structure of mofules...

So from my point of view as the "docs maintainer".. and also using "master" for projects..

Is that I want to sync them..

First step would be to please after all this time..
merge the "develop" NOW into master
and then I can go and hack away at the "next" version coming downstream..

We need more community, more throw it all in, and see if it crashes.. and more fun
rather than the slower "stable" approach..

were not at 1.0 yet,,
and end users so  far have their stuff.. (binary or code)

so we are allowed AFAIK to actually really mess up things..
and this is what we want..
make it crash and a message to upgrade..
and read the api docs.... (which I maintain.. along with everyone)......

Infact bump the version, penguin style
to  0.2.0 or alike.. 

Anyway.. am off to bed..
and looking forward to hacking away tomorrow morning..
the lunar eclipse her in wales starts at
8.20 am
Peak - 9.32 am
and end = 10.32

Looking forward

**Comment by brendensoares** [comment]
I removed @landaire.

@notzippy I like your roles. It's a great idea, but I'm concerned about people sticking to it. I'd love to define these roles in our wiki and use them for v0.13 and see how it goes!

Also, @anonx, I asked our community to use StackOverflow, not our forums. I want to increase Revel's exposure to the general dev community and I don't think a mailing list is the right location to do that. Either way, we do need help staying on top of community activity.

I added it to my todo list to catch up on the mailing list.

I am still interested in Revel and do my best despite other obligations.

@notzippy we have already decided to make reorganization ongoing isntead of blocking further development. @anonx makes good points about not ignoring developer needs over internal organization and code beauty. So, point is, let's keep making forward progress and continuous improvement together.

We use to have a weekly team meeting, but people got busy and it was hard to maintain. We also now have to consider timezones as @pedromorgan is on GMT and the rest of us are in US timezones.

There are many challenges to face. Some of the biggest is **prioritizing** the community's requests with our own and then **implementing** it properly with tests to verify all is as expected.


@pedromorgan I am strongly against using `master`. It does not work out well. Too many fires to put out in that workflow. I agree we need more frequent releases, though. I already mentioned on a RFC repo issue that the RFCs are  there to define major changes, like the asset pipeline, not bug fixes and small, obvious enhancements. I feel there is value in planning with RFCs.

My biggest goal is getting the weekly meeting going again with our team. From there we can discuss further. long threads like this one can be resolved much faster in real time, and preferably via voice instead of text. **Can we do this weekly**?

PS - @anonx I've invited more contributors to join core team (we'll see who joins):
- @jgraham909
- @tmbrggmn
- @iamjem
- @anonx 
- @klauspost 
- @ottob
